mixer/pkg/api/grpcServer.go

```diff
@@ -230,6 +367,83 @@ func (s *grpcServer) check(ctx context.Context, req *mixerpb.CheckRequest,
 	return resp, nil
 }
 
+
+//Access log service shold return empty response
+func (s *grpcServerEnvoy) StreamAccessLogs(
```

Comment:
nit: we should clean up the comments to discuss how this is implementing the Envoy ALS service, etc.

Reply:
done